### PR TITLE
Create io.github.prateekmedia.pstube

### PR DESCRIPTION
**Pull request by prateekmedia**
This app is the successor of [FluTube](https://github.com/flathub/io.github.prateekmedia.flutube), which is now renamed to PsTube. I have made flutube end-of-life in flathub repo.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by prateekmedia** [comment]
bot build io.github.prateekmedia.pstube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.pstube

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "skip-arches": [
+        "aarch64"
```
Comment (hfiguiere) [inline flathub.json:3]: :-(

File: io.github.prateekmedia.pstube [hunk 2]
```diff
@@ -0,0 +1,40 @@
+app-id: io.github.prateekmedia.pstube
```
Comment (hfiguiere) [inline io.github.prateekmedia.pstube:1]: this file is improperly named

Comment (prateekmedia) [reply io.github.prateekmedia.pstube:1]: Whats the proper one then?

File: flathub.json (hunk 1 above)
Comment (prateekmedia) [reply flathub.json:3]: i still dont know any exisiting project that use Flutter to build here. I have taken inspiration writing this template too from FluffyChat so I dont know much about Building from source.

File: io.github.prateekmedia.pstube (hunk 2 above)
Comment (prateekmedia) [reply io.github.prateekmedia.pstube]: Oh got it, I was just trying copy paste from previous one and missed it.

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.pstube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.pstube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.pstube

**Comment by barthalion** [comment]
About FluTube, consider changing end-of-life and adding end-of-life-rebase to match https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life= once this is published.

/merge

**Pull request closed without merging**